packages/gasket-utils/lib/lazy-load-package.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs/promises');
+const path = require('path');
+const PackageManager = require('./package-manager');
+const tryResolve = require('./try-resolve');
+
+/**
+ * Determine package manager
+ * @param {string} root Gasket root
+ * @param {Logger} logger Gasket logger
+ * @returns {string} Package manager cmd
+ */
+async function getPkgManager(root, logger) {
+  const yarnLock = await fs.readFile(path.join(root, 'yarn.lock'), 'utf8').catch(() => {
+    logger.info('LazyLoadPackage - Installing using npm');
+  });
+
+  if (yarnLock) {
+    logger.info('LazyLoadPackage - Installing using yarn');
+    return 'yarn';
+  }
+
+  return 'npm';
+}
+
+/**
+ * Lazy load package - load devDependency programmatically when needed
+ * @param {string} dependency The require'ed dep needed
+ * @param {Gasket} gasket Gasket instance
+ * @returns {object} module
+ */
+module.exports = async function lazyLoadPackage(dependency, gasket) {
+  const { logger } = gasket;
+  const { root } = gasket.config;
+  const cmd = await getPkgManager(root, logger);
+  const pkg = dependency.split('/')[0];
+  const modulePath = tryResolve(dependency, [root, __dirname]);
+
+  if (modulePath) {
+    logger.info(`LazyLoadPackage - Package "${dependency}" already installed`);
+    return require(dependency);
+  }
+
+  logger.info(`LazyLoadPackage - Package "${dependency}" not found`);
+  logger.warning(`LazyLoadPackage - installing "${pkg}" - save as a devDependency to avoid this`);
```

Comment:
This function is overly verbose right now. Let's get down to one message.

We really only need to inform the user what is going on when it appears slow (it's installing) and to let them know how to avoid it in the future.

You can let them know the package manager being used in this single log message as well.

```suggestion
  logger.info(`LazyLoadPackage - ${cmd} installing "${pkg}" - save as a devDependency to avoid this`);
```